libs/ui-lib/lib/common/components/clusterConfiguration/ControlPlaneNodesDropdown.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup, Tooltip } from '@patternfly/react-core';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+import { getFieldId, StaticField } from '../..';
+import { useField } from 'formik';
+
+interface ControlPlaneNodesOption {
+  value: string;
+  label: string;
+}
+
+const isDropdownItemEnabled = (controlPlaneNodeCount: string, isNutanix?: boolean): boolean => {
+  return (controlPlaneNodeCount === '1' && !isNutanix) || controlPlaneNodeCount !== '1';
+};
+
+const ControlPlaneNodesDropdown = ({
+  isDisabled = false,
+  isNutanix,
+}: {
+  isDisabled?: boolean;
+  isNutanix?: boolean;
+}) => {
+  const { t } = useTranslation();
+  const [{ name, value }, , { setValue }] = useField<string>('controlPlaneAgents');
+  const [current, setCurrent] = React.useState<string>('3');
+
+  const options: ControlPlaneNodesOption[] = [
+    { value: '1', label: t('ai:1 (Single Node OpenShift - not highly available cluster)') },
+    { value: '3', label: t('ai:3 (highly available cluster)') },
+    { value: '4', label: t('ai:4 (highly available cluster+)') },
+    { value: '5', label: t('ai:5 (highly available cluster++)') },
+  ];
+
+  const dropdownItems = options.map(({ value, label }) => {
+    const isItemEnabled = isDropdownItemEnabled(value, isNutanix);
```

Comment:
Why do we check specifically for Nutanix ? what about Vsphere / other platforms ?

Reply:
in CIM it seems that for Nutanix is not supported SNO.
@danmanor the APIs are different, how can I check if 4/5 CPN are supported?

Reply:
My bad forgot it is RestAPI only. No other way but hardcoding it currently